Framework/Core/include/Framework/DataOutputDirector.h

```diff
@@ -17,6 +17,10 @@
 #include "Framework/DataSpecUtils.h"
 #include "Framework/InputSpec.h"
 
+#include "rapidjson/document.h"
+#include "rapidjson/filereadstream.h"
+
+#include <cstdio>
```

Comment:
Please move the code using this inside the implementation, not in the header.

Reply:
done